src/Appwrite/Usage/Calculators/Aggregator.php

```diff
@@ -0,0 +1,229 @@
+<?php
+
+namespace Appwrite\Usage\Calculators;
+
+use Utopia\Database\Database as UtopiaDatabase;
+use Utopia\Database\Document;
+use Utopia\Database\Query;
+
+class Aggregator extends Database
```

Comment:
Is this supposed to extend `Database` ? Shouldn't it extend the `Calculator` class ?